iris.script

```diff
@@ -1,10 +1,18 @@
-    ; run installer to create namespace
-    do $SYSTEM.OBJ.Load("/opt/irisapp/Installer.cls", "ck")
-    set sc = ##class(App.Installer).setup() 
+ ; add a bit of stdandard docker comfort
+ zn "%SYS"
+ set sc=##class(Security.Users).UnExpireUserPasswords("*")
+ w sc hang 15
+ set user=##class(Security.Users).%OpenId("unknownuser")
+ do user.Roles.Insert("%All")
```

Comment:
why do you add %All here? I don't see a need to override the container's default security setup beyond unexpiring the password

Reply:
That's my personal approach: 
- if I'm testing FEATURES and FUNCTIONALITY I have no motivation to fight around security. with no essential needs.
- take it out if you dislike it. I'll put it in my test scenarios anyhow manually at every test I run

Reply:
That's my personal approach:

if I'm testing FEATURES and FUNCTIONALITY I have no motivation to fight around security. with no essential needs.
take it out if you dislike it. I'll put it in my test scenarios anyhow manually at every test I run

It's now uncommented